spec/models/note_spec.rb

```diff
@@ -0,0 +1,4 @@
+require 'rails_helper'
+
+RSpec.describe Note, type: :model do
+end
```

Comment:
I think if we stick with the has_many approach, it would be great to make sure we don't allow them to save empty notes.